### DISCUSSION TITLE
Linkerd on Minikube

### DISCUSSION THREAD

**Discussion by akilans** [linkerd/linkerd2#5039] [General] [answered]
Hi Everyone,

I am trying to create Linkerd POC in minikube setup.

#### Setup
```
minikube - v1.13.1
# kubectl - 
Client Version: v1.18.5
Server Version: v1.19.2
# Linkerd
Client version: stable-2.8.1
Server version: stable-2.8.1
```

``` bash
linkerd check
# Shows success √ 
```
But when I try to inject sidecar proxy by running the below command it is not injecting sidecar proxy
```
kubectl get deployment -o yaml | linked inject - | kubectl apply -f -
```

Do I need to enable any addons in minikube?. 

Thanks in advance
Akian

**Comment by grampelberg** [comment, marked as answer]
See #4945. This is fixed in edge and will be stable as part of 2.9.

**Reply by akilans** [reply]
Thanks for your response @grampelberg. Yea, i downgraded my k8s to 1.18. it worked.
